linker.go

```diff
@@ -457,3 +458,38 @@ func resolveKconfigReferences(insns asm.Instructions) (_ *Map, err error) {
 
 	return kconfig, nil
 }
+
+func resolveKsymReferences(insns asm.Instructions) error {
+	var missing []string
+
+	iter := insns.Iterate()
+	for iter.Next() {
+		ins := iter.Ins
+		meta, _ := ins.Metadata.Get(ksymMetaKey{}).(*ksymMeta)
+		if meta == nil {
+			continue
+		}
+
+		v := meta.Var
+		if v.Addr != 0 {
+			ins.Constant = int64(v.Addr)
+			continue
+		}
+
+		if meta.Binding == elf.STB_WEAK {
```

Comment:
AFAIK since this anyway is a simple constant load operation, we can just set te constant to `0`. I guess `0` should be fine, as this is also what libbpf seems to be doing ([link](https://github.com/torvalds/linux/blob/v5.15/tools/lib/bpf/libbpf.c#L6857)).

I will add a comment though, haha.

Reply:
Agreed that it's good for clarity, thanks for the link!